packages/platform-sdk-profiles/src/drivers/memory/wallets/wallet.factory.ts

```diff
@@ -210,10 +212,12 @@ export class WalletFactory implements IWalletFactory {
 	}
 
 	private allowsDeriveWithBIP49(wallet: IReadWriteWallet): boolean {
+		/* istanbul ignore next */
```

Comment:
Not used yet so I won't bother with coverage. Will need implementing for Bitcoin.